internal/service/dns_config/model_config_view.go

```diff
@@ -440,21 +439,20 @@ var ConfigViewResourceSchemaAttributes = map[string]schema.Attribute{
 	},
 	"zone_authority": schema.SingleNestedAttribute{
```

Comment:
Did the zone_authority have a problem with `view` I thought it was just for zones?

Reply:
zone authority has the same issue if default isnt added for all values , which is technically incorrect

Reply:
In API, they recently added `ns.b1ddi` and `hostmaster` as defaults for `mname` and `rname`, so you can add those default values

Reply:
fixed